requirements.txt

```diff
@@ -1,4 +1,5 @@
 django==4.2.10
+django-cache-memoize==0.2.0
```

Comment:
Don't use cache-memoize, standard library should be enough

Reply:
Ok, is `gnosis.util.cache` correct here?

Reply:
Yes, it's a wrapper around `functools.cache` as it was added in Python 3.9 : https://docs.python.org/3/library/functools.html#functools.cache